README.md

```diff
@@ -1,23 +1,28 @@
 # Kubecon-India-2024-sponsorship
+
 ![Image](https://github.com/user-attachments/assets/e1e03137-31c6-4322-bee6-d4d8e7f4be57)
 
-* Hello folks 👋
-* We are glad to announce that we are running this programme to sponsorship the KubeCon-India-2024 tickets.
---------
+- Hello folks 👋
+- We are glad to announce that we are running this programme to sponsorship the KubeCon-India-2024 tickets.
```

Comment:
Can we change sponsorship to sponsor here ?

Reply:
Yes sure